homeassistant/components/plex/button.py

```diff
@@ -0,0 +1,67 @@
+"""Representation of Plex buttons."""
+from __future__ import annotations
+
+import datetime
+
+from homeassistant.components.button import ButtonEntity
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.dispatcher import async_dispatcher_send
+from homeassistant.helpers.entity import DeviceInfo, EntityCategory
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.event import async_track_time_interval
+
+from .const import CONF_SERVER_IDENTIFIER, DOMAIN, PLEX_UPDATE_PLATFORMS_SIGNAL
+
+CLIENT_SCAN_INTERVAL = datetime.timedelta(minutes=10)
+
+
+async def async_setup_entry(
+    hass: HomeAssistant,
+    config_entry: ConfigEntry,
+    async_add_entities: AddEntitiesCallback,
+) -> None:
+    """Set up Plex button from config entry."""
+    server_id: str = config_entry.data[CONF_SERVER_IDENTIFIER]
+    async_add_entities([PlexScanClientsButton(server_id)])
+
+
+class PlexScanClientsButton(ButtonEntity):
+    """Representation of a scan_clients button entity."""
+
+    _attr_entity_category = EntityCategory.CONFIG
+    _attr_name = "Scan Clients"
```

Comment:
You don't want to add the Plex server name or something to distinguish the entity?

Reply:
Adjusted.